README.md

```diff
@@ -343,6 +347,15 @@ Found '/path/to/project/.nvmrc' with version <5.9>
 Now using node v5.9.1 (npm v3.7.3)
 ```
 
+`nvm use` et. al. will traverse directory structure upwards from the current
+directory looking for the `.nvmrc` file. In other words, running `nvm use`
+et. al. in any subdirectory of a directory with an `.nvmrc` will result in that
+`.nvmrc` being utilized.
+
+The contents of a `.nvmrc` file **must** be the node version number
+or specification followed by a newline. No trailing spaces are allowed
```

Comment:
"or specification" isn't very clear; `nvm --help` describes it as a "version-ish"; another way to phrase it is "must be the exact argument passed to `nvm use` or other `nvm` commands, followed by a newline"

Reply:
While this is technically correct I don't think it is helping newcomers understand what goes into an nvmrc.

Reply:
Do you think newcomers will understand "specification" when it's not a term that's used anywhere else in docs?

Reply:
Feel free to make any changes you want, I will not be offended.